libs/barista-components/tile/src/chart-colors.ts

```diff
@@ -1,82 +0,0 @@
-/**
```

Comment:
why was that one in the tile at all? 
very weird.

Reply:
I don't know very strange.

Reply:
💯 + for bazel. 
I'm awaiting the day, when it will do laundry and dishes for me 😜